frodo.js

```diff
@@ -1,75 +1,80 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!